hudi-client/src/test/java/org/apache/hudi/common/HoodieTestDataGenerator.java

```diff
@@ -18,9 +18,15 @@
 
 package org.apache.hudi.common;
 
+import static org.apache.spark.sql.types.DataTypes.*;
```

Comment:
Please split the * into seperate import.

Reply:
Done